hw/ip/rbox/data/rbox.hjson

```diff
@@ -0,0 +1,245 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "rbox",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  available_input_list: [
+    { name: "ac_present", desc: "A/C power is present" }
+    { name: "ec_entering_rw", desc: "Embedded controller is entering the R/W mode in the boot flow" }
+    { name: "key0_in", desc: "VolUp button in tablet; column output from the EC in a laptop" }
+    { name: "key1_in", desc: "VolDown button in tablet; row input from keyboard matrix in a laptop" }
+    { name: "key2_in", desc: "TBD button in tablet; row input from keyboard matrix in a laptop" }
+    { name: "pwrb_in", desc: "Power button in both tablet and laptop" }
+  ],
+  available_output_list: [
+    { name: "bat_en", desc: "Battery is enabled" }
+    { name: "ec_in_rw", desc: "Embedded controller is in the R/W mode. It’s a flopped version of ec_entering_rw" }
+    { name: "ec_rst_l", desc: "Asserted when por_l(Power On Reset) is low. Released a short period after por_l is high" }
+    { name: "flash_wp_l", desc: "Write protection. Active low. :Asserted when por_l is low. Released by FW" }
+    { name: "key0_out", desc: "Passthrough from key0_in, can be configured to invert" }
+    { name: "key1_out", desc: "Passthrough from key1_in, can be configured to invert" }
+    { name: "key2_out", desc: "Passthrough from key2_in, can be configured to invert" }
+    { name: "pwrb_out", desc: "Passthrough from pwrb_in, can be configured to invert" }
+  ],
```

Comment:
General question here:

Could you categorize the signals into two groups? One for actual chip in/out others for AST connection. Are all chip input/output? If yes, can all signals be multiplexed IO (shared with some other modules) or dedicated one? I don't remember the detail but I think all of them are dedicated iO?